CHANGES.md

```diff
@@ -50,6 +50,15 @@
 * ([#X](https://github.com/apache/beam/issues/X)).
 -->
 
+# [2.44.0] - Unreleased
+
+## New Features / Improvements
```

Comment:
Can we also add a unit test for this feature ?

Reply:
Done.